### PR TITLE
Restore this.getNativeLanguages() in NewStudent.vue

### PR DESCRIPTION
**Pull request by therealyingtong**
On current master, the Native Languages autocomplete on the New Student form does not work. I believe this bug was introduced here: https://github.com/TechLadies/irrc-madwish/pull/136/files#diff-c52fea3e6e55697a969968cce9ca59c1f8b10e21bbd297d5d4788b1097d503a0L145

To test:
- go to New Student page
- start typing in the Native Language field. Check that autocomplete options show up for existing languages (e.g. "Hindi")

NB: This has #141 merged into it. #141 already had two approving reviews.

### CODE REVIEW

**Review by dzldebby — APPROVED** [review]
LGTM

File: frontend/src/pages/NewStudent.vue [hunk 1]
```diff
@@ -175,17 +177,14 @@ export default {
...
-          this.$buefy.snackbar.open({
-            message: "New Student Added",
-            type: "is-success",
-            position: "is-top",
-            actionText: "View Profile",
-            indefinite: true,
-            onAction: () => {
-              this.$router.push({ path: `/students/${data.StudentID}` });
-            },
-          });
```
Comment (thchia) [inline frontend/src/pages/NewStudent.vue]: As discussed, this should not have been removed.

**Review by thchia — CHANGES_REQUESTED** [review]

File: frontend/src/pages/NewStudent.vue (hunk 1 above)
Comment (therealyingtong) [reply frontend/src/pages/NewStudent.vue]: Restored.

**Review by thchia — APPROVED** [review]
Untested, but LGTM. I am wondering if it's better to put the Native Language refetch in the Vuex action. That way, no matter where you call the `createStudent` or `createTeacher` from, you don't need to keep remembering to refetch the Native Languages. But not a blocking concern.

**Review by dzldebby — APPROVED** [review]
Tested, works well!

**Pull request merged**